documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -121,6 +121,16 @@
                      "counter":0,
                      "roughness":0.0
                   },
+                  "fetched_versions":{
+                     "hz":0.0,
+                     "counter":0,
+                     "roughness":0.0
+                  },
+                  "fetch_frequency":{
```

Comment:
Because this is reporting an entire counter, we can name it to match the event being counted, such as:

```suggestion
                  "fetches_from_logs":{
```

With that, `fetches_from_logs.hz` would be the fetch frequency.

Reply:
Yes, makes sense.